script/DeployEverlongStrategy.s.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.24;
+
+import { IHyperdrive } from "hyperdrive/contracts/src/interfaces/IHyperdrive.sol";
+import { IEverlongStrategy } from "../contracts/interfaces/IEverlongStrategy.sol";
+import { EVERLONG_STRATEGY_KIND, EVERLONG_STRATEGY_KEEPER_KIND } from "../contracts/libraries/Constants.sol";
+import { EverlongStrategy } from "../contracts/EverlongStrategy.sol";
+import { BaseDeployScript } from "./shared/BaseDeployScript.sol";
+
+/// @title DeployEverlongStrategy
+/// @notice EverlongStrategy deployment script.
+/// @custom:disclaimer The language used in this code is for coding convenience
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract DeployEverlongStrategy is BaseDeployScript {
```

Comment:
Have you tested verification with these scripts? That was the main reason that we moved away from Foundry scripts. These contracts are simpler to deploy that Hyperdrive's, but it would be good to check.

Reply:
I tested them as far as the scripts working... not sure if there's a different context you're thinking of